apps/package.json

```diff
@@ -302,6 +302,7 @@
     "react-portal": "^4.2.1",
     "react-router-dom": "^4.3.1",
     "react-transition-group": "2.9.0",
+    "react-typist": "^2.0.5",
```

Comment:
When a new package is added, do we let infra know?

Reply:
Not in my experience.  This is a fairly regular occurrence.